base/applications/regedit/regproc.c

```diff
@@ -851,9 +851,15 @@ static void processRegLinesW(FILE *in)
                 if(*s_eol == '\r' && *(s_eol+1) == '\n')
                     NextLine++;
 
-                while(*(NextLine+1) == ' ' || *(NextLine+1) == '\t')
+                while(*(NextLine) == ' ' || *(NextLine) == '\t')
```

Comment:
Remove the extra parentheses around "NextLine".
I'm also wondering whether one can replace this with a `while (isspace(*NextLine))` (or use isspaceW in case the strings are unicode).

Reply:
Yes, it's also works well. Commit updated